Update shoulder-surfing@FD0bkmxNpPXiUB_NevEUf.md (#6708)
kamranahmedse/developer-roadmap: src/data/roadmaps/cyber-security/content/shoulder-surfing@FD0bkmxNpPXiUB_NevEUf.md

> # Shoulder Surfing
> Shoulder Surfing Attack in which attacker try to get information when you are unaware where attacker looks over your shoulder or from your back to see what you're doing on your device and obtain sensitive information.Shoulder surfing attacks are accomplished by observing the content "over the victim's shoulder".It is a social engineering attack where attacker physically view the device screen and keypad to obtain personal information and this attack mostely done when you are at public place or in crowded area.Sometime attacker attack when you are busy in your device and attacker could be your friend, someone you know or it may be some stranger.
> 
> Visit the following resources to learn more:
> - [@article@What is Shoulder Surfing](https://www.geeksforgeeks.org/what-is-shoulder-surfing-in-cyber-security/)
> - [@article@What is shoulder surfing, and how can you avoid it?](https://nordvpn.com/blog/shoulder-surfing/?srsltid=AfmBOorl5NPpW_Tnhas9gB2HiblorqwXyK0NJae7uaketrnDwbjJmiYV)
> - [@article@What is Shoulder Surfing?](https://www.mcafee.com/learn/what-is-shoulder-surfing/)
> - [@article@What is Shoulder Surfing? 9 ways to protect yourself](https://www.bigrock.in/blog/products/security/what-is-shoulder-surfing-9-ways-to-protect-yourself-from-shoulder-surfing/)

Reviewer: ```suggestion
# Shoulder Surfing

In a Shoulder Surfing Attack, an attacker tries to get information when you are unaware of where the attacker looks over your shoulder or from your back to see what you're doing on your device and obtain sensitive information. Shoulder Surfing attacks are accomplished by observing the content "over the victim's shoulder". It is a social engineering attack where the attackers physically view the device screen and keypad to obtain personal information. This attack is mostly done when you are in a public place or crowded area. Sometimes attackers attack when you are busy on your device and the attacker could be your friend, someone you know or it may be some stranger.

Visit the following resources to learn more:

- [@article@What is Shoulder Surfing](https://www.geeksforgeeks.org/what-is-shoulder-surfing-in-cyber-security/)
- [@article@What is shoulder surfing, and how can you avoid it?](https://nordvpn.com/blog/shoulder-surfing/?srsltid=AfmBOorl5NPpW_Tnhas9gB2HiblorqwXyK0NJae7uaketrnDwbjJmiYV)
- [@article@What is Shoulder Surfing?](https://www.mcafee.com/learn/what-is-shoulder-surfing/)
- [@article@What is Shoulder Surfing? 9 ways to protect yourself](https://www.bigrock.in/blog/products/security/what-is-shoulder-surfing-9-ways-to-protect-yourself-from-shoulder-surfing/)

```
